### PR TITLE
Rename extension

### PR DESCRIPTION
**Pull request by slifty**
In order to install SimpleBook in ansible we want to remove references to things like `Collection.php` etc.

This is a first set of renames which should allow for SimpleBook installation.

Note that SimpleBook and Collection should NOT be installed on the same mediawiki instance at this point, since they share some of the same configuration variable names.

Issue #25

### CODE REVIEW

**Comment by slifty** [comment]
I'm actually just gonna merge this (can handle any feedback later) since this is the first step of several and this actually fixes the ability to deploy to devenv via ansible.

File: SimpleBook.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * Collection Extension for MediaWiki
+ * SimpleBook Extension for MediaWiki
  *
  * Copyright (C) PediaPress GmbH
```
Comment (hawkinsw) [inline SimpleBook.php:6]: Sorry to drop in late. Do we need to update this?

**Review by hawkinsw — COMMENTED** [review]
LGTM!

File: SimpleBook.php (hunk 1 above)
Comment (slifty) [reply SimpleBook.php:6]: I was wondering and figured I'd ask @kfogel that some time / do that on a future PR if so.

Hey @kfogel do we need to update that?

Comment (kfogel) [reply SimpleBook.php:6]: Yes, we should update that.  Now, we should also be clearly documenting (which I think we are?) in the top-level README that this is descended from Collection.  We want to be sure to keep provenance and proper attribution and all that.

Also when we modify a file, we should update the Copyright line to name OTS as well, e.g.,:

`Copyright (C) PediaPress GmbH; Open Tech Strategies, LLC`

Comment (kfogel) [reply SimpleBook.php:6]: (For any new files that we've added, the copyright line should just name OTS, of course.)

Comment (slifty) [reply SimpleBook.php:6]: Oh good to know!

On a related note, I don't usually include Copyright lines in *any* files (on any project...)

Is that... is that a mistake?

image

Comment (kfogel) [reply SimpleBook.php:6]: Linking in our concurrent [Zulip conversation about this](https://chat.opentechstrategies.com/#narrow/stream/2-general/topic/All.20Things.20open/near/111680).

**Pull request merged**